src/shared/creds-util.c

```diff
@@ -656,6 +656,7 @@ int encrypt_credential_and_warn(
 
 #if HAVE_TPM2
         bool try_tpm2;
+        Tpm2Support support;
```

Comment:
move this declaration inside the block where it's used

Reply:
Should be done